src/frontend/packages/input-components/src/ImageInput.tsx

```diff
@@ -38,36 +32,24 @@ const format = (v: undefined | PreFormatValue | PreFormatValue[]) => {
 };
 
 type RawInput = File | { src: string };
-type ParsedInput = AddedFile | DeletedFile | string | null;
-
-const parse =
-  (oldValue: string | null) =>
-  (v: RawInput | RawInput[] | null): ParsedInput | ParsedInput[] => {
-    if (Array.isArray(v)) {
-      return v.map(e => parse(oldValue)(e) as ParsedInput);
-    }
-    if (v instanceof File) {
-      return transformFile(v, oldValue);
-    }
-    if (v && v.src) {
-      return v.src;
-    }
-    if (!v && oldValue) {
-      return {
-        fileToDelete: oldValue
-      };
-    }
-
-    return null;
-  };
+type ParsedInput = AddedFile | string | null;
 
-type Props = ImageInputProps;
+const parse = (v: RawInput | RawInput[] | null): ParsedInput | ParsedInput[] => {
+  if (Array.isArray(v)) {
+    return v.map(e => parse(e) as ParsedInput);
+  }
+  if (v instanceof File) {
+    return transformFile(v);
+  }
+  if (v?.src) {
+    return v.src;
+  }
```

Comment:
Before your other PR, this line was slightly different:
https://github.com/assemblee-virtuelle/semapps/blob/820bf842d905315ee2da922e61e6dc8a39c9bd40/src/frontend/packages/input-components/src/ImageInput.js#L29-L31
Could you use the same code in case it was the cause of the bug in the Oasis instance ?